### PR TITLE
docs: update VGV logo

### PR DESCRIPTION
**Pull request by tomarra**
## Description

Updating the VGV logo in the Readme.

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,11 +166,14 @@ Update the `CFBundleLocalizations` array in the `Info.plist` at `ios/Runner/Info
...
 [very_good_analysis_badge]: https://img.shields.io/badge/style-very_good_analysis-B22C89.svg
 [very_good_analysis_link]: https://pub.dev/packages/very_good_analysis
 [very_good_cli_link]: https://github.com/VeryGoodOpenSource/very_good_cli
 [very_good_ventures_link]: https://verygood.ventures/?utm_source=github&utm_medium=banner&utm_campaign=core
 [event_link]: https://dsc.community.dev/events/details/developer-student-clubs-polytechnic-university-of-milan-presents-flutter-very-good-app/
 [dsc_link]: https://dsc.community.dev/polytechnic-university-of-milan/
 [slides_link]: slides/recipe_for_a_very_good_app.pdf
+[logo_black]: https://raw.githubusercontent.com/VeryGoodOpenSource/very_good_cli/main/assets/images/vgv_logo_black.png#gh-light-mode-only
+[logo_white]: https://raw.githubusercontent.com/VeryGoodOpenSource/very_good_cli/main/assets/images/vgv_logo_white.png#gh-dark-mode-only
+[very_good_ventures_link_dark]: https://verygood.ventures/?utm_source=github&utm_medium=banner&utm_campaign=CLI#gh-dark-mode-only
```
Comment (felangel) [inline README.md:178]: Can we remove the utm tags in the url?

File: README.md [hunk 2]
```diff
@@ -165,11 +166,14 @@ Update the `CFBundleLocalizations` array in the `Info.plist` at `ios/Runner/Info
...
 [license_link]: https://opensource.org/licenses/MIT
-[logo]: https://raw.githubusercontent.com/VeryGoodOpenSource/very_good_analysis/main/assets/vgv_logo.png
 [very_good_analysis_badge]: https://img.shields.io/badge/style-very_good_analysis-B22C89.svg
 [very_good_analysis_link]: https://pub.dev/packages/very_good_analysis
 [very_good_cli_link]: https://github.com/VeryGoodOpenSource/very_good_cli
 [very_good_ventures_link]: https://verygood.ventures/?utm_source=github&utm_medium=banner&utm_campaign=core
 [event_link]: https://dsc.community.dev/events/details/developer-student-clubs-polytechnic-university-of-milan-presents-flutter-very-good-app/
 [dsc_link]: https://dsc.community.dev/polytechnic-university-of-milan/
 [slides_link]: slides/recipe_for_a_very_good_app.pdf
+[logo_black]: https://raw.githubusercontent.com/VeryGoodOpenSource/very_good_cli/main/assets/images/vgv_logo_black.png#gh-light-mode-only
```
Comment (felangel) [inline README.md:176]:
```suggestion
[logo_black]: https://raw.githubusercontent.com/VGVentures/spacex_demo/main/assets/images/vgv_logo_black.png#gh-light-mode-only
```

Comment (tomarra) [reply README.md]: Fixed

File: README.md (hunk 1 above)
Comment (tomarra) [reply README.md]: Good call. Removed.

**Review by felangel — APPROVED** [review]
🎉

**Pull request merged**